eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheetSlot.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import Table from "@material-ui/core/Table";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { CustomerWithAttendance, SlotInterface } from "@/types/temp";
+
+import { getSlotTimespan } from "@/utils/helpers";
+
+/** @TODO remove this and import same type when it's merged*/
+export interface Props extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
+
+const AttendanceSheetSlot: React.FC<Props> = ({
+  customers,
+  intervals,
+  type,
+  notes,
+}) => {
+  const classes = useStyles();
+
+  const timeString = getSlotTimespan(intervals);
+
+  /** UI todos:
+   * @TODO in page component render alternative background shades for slot table
+   * @TODO center each slot timestring
+   * @TODO justify spaces in rows with 2 columns (name cell is much bigger than signature cell)
+   *
+   */
+  return (
+    <Table aria-label="simple table">
+      <TableHead className={classes.timeString}>
+        <TableRow>
+          <TableCell>
+            {timeString} {type.toUpperCase()}
+            {notes}
+          </TableCell>
+        </TableRow>
+      </TableHead>
+      <TableBody className={classes.tableRow}>
+        {Object.keys(intervals).map((interval) => (
+          <React.Fragment key={interval}>
+            <TableRow>
```

Comment:
The same as before: Rather assign a `tableRow` class to each `TableRow` than assigning it to a container and targeting nested table rows.